examples/python/ToF/tof_raw_rvc4.py

```
@@ -0,0 +1,244 @@
+        ax.axis("off")
+        fig.colorbar(im, ax=ax, fraction=0.046, pad=0.04)
+
+    fig.suptitle("Replayed ToF depth (from saved raw frames)")
+    fig.tight_layout()
+    fig.savefig(out_path, dpi=150)
+    print(f"[Plot]   Saved to: {out_path}")
+
+    try:
+        plt.show()
+    except Exception:
+        pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)